Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -21,12 +22,20 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
      * Return the time remaining to when the battery is charged or depleted*/
     QTime timeWhenChargedOrDepleted() const;
 
+    int getRemainingHours() const;
+
     /*Update Variables*/
     void addData(const BatteryData& batteryData);
+    void setRemainingHours(int h);
```

Comment:
Change "h" to something more descriptive.

Reply:
setRemainingHours was removed in a later commit